docs/how-to/publishing/android.rst

```diff
@@ -0,0 +1,132 @@
+configuration in the store. Finally, you will create a new release of the app,
+at which point you will upload the APK files that comprise your app.
+
+Creating a listing
+++++++++++++++++++
+
+To distribute your app on the Google Play Store, you need to visit
+https://play.google.com/apps/publish/ and create an account.
+
+Once you've done that, click **Create Application**. Choose a language and
+write a brief app title, up to 50 characters. This should probably be the
+same as your app's Formal Name in the BeeWare ecosystem.
+
+This will take you to **Store Listing** section of your app. You will need to
+answer a variety of questions. Take your time to answer them well.
+
+You will need to upload an icon. For briefcase apps, an icon is typically specified in
+your app's ``pyproject.toml`` file. Look for a line starting with ``icon = ``.
+The Google Play Store will require a variety of other information, including at least
+one screenshot of your app and your assessment of likelihood that children will want
+to use the app.
+
+Configure app signing
++++++++++++++++++++++
+
+Click **App Signing** in the left navigation. The Google Play Store offers an app signing
+feature where they maintain the signing key for your app. For simplicity, you will
+click **Opt Out** to opt out of that feature.
+
+.. admonition:: Letting Google manage and protect your app signing key
+
+  The Google Play store's feature to manage app signing keys offers a way to
+  protect your app if your signing key is ever lost or compromised. You can read
+  more about it in `Google's documentation. <https://support.google.com/googleplay/android-developer/answer/7384423?hl=en>`_
+  For simplicity, this documentation relies on you to manage and protect your
+  app's signing key.
+
+  You can enable this feature later if you are willing to give Google the private
+  key corresponding to this app.
+
+Creating a release and uploading APK files
+++++++++++++++++++++++++++++++++++++++++++
+
+To create a release, click **Release management** in the navigation.
+Choose **App releases.**  **Production**, click the button labeled
+**Manage.** In this section, click **Create Release.**
+
+If prompted to enable Google's hosted app signing, click **Opt Out**. You may have
+to click **I'm Sure, Opt Out** for now. If you do prefer to set up Google's hosted
+app signing, you may configure that, but it is beyond the scope of this document.
+
+Scroll to **Android App Bundles and APKs to add** and click **Browse Files.** This will
+allow you to choose the four APK files under the TODO directory.
+
+Scroll to **What's new in this release?**. If this is your first upload of the app,
+write the words, "Initial upload."
+
+Finally, click **Review.**
+
+This will prompt a variety of questions from the Google Play Store about the content
+of your app, including the absence/presence of advertising, its appropriateness for
+different age groups, and any embedded commercial aspects.
```

Comment:
I also had to save a draft, then visit the "Content Rating", "App Content" and "Pricing and Distribution"  tabs to fill out all sorts of details. There's a series of grayed out arrows; until they're all green, you can't publish. There wasn't a "just answer the questions" workflow.

It also raised a warning about being a non-optimized bundle. Short term, we need to flag this isn't a problem; long term, we should modify the publish command to produce multiple APKs.

Reply:
Fixed now. (Also, the Android App Bundle saves us from the non-optimized bundle problem! Yay.)